### ISSUE TITLE
DropdownList 'disabled' assumes array.

### ISSUE THREAD

**Issue by cyrusbowman** [jquense/react-widgets#489]
Tested on v3.4.4

```
<DropdownList className={styles.dropdown} textField='label' valueField='value' 
    data={[{label: 'Yes', value: true}, {label: 'No', value: false}]}
    value={true}
    disabled={false} />
```

Result: 'No' is disabled in the dropdown.
Expected: All the values in the list to be enabled.

In this case if I wanted 'No' to be disabled I would expect to do this:

```
<DropdownList className={styles.dropdown} textField='label' valueField='value' 
    data={[{label: 'Yes', value: true}, {label: 'No', value: false}]}
    value={true}
    disabled={[false]} />
```

**Comment by jquense** [comment]
you can just leave `disabled` `undefined` or `null` but I agree that the current behavior is a bug

**Comment by cyrusbowman** [comment]
Fix included in pull request #490

**Comment by jquense** [comment]
fixed in the upcoming v4. try out the betas
